Wikipedia/Code/WMFArticlePreviewTableViewCell.m

```diff
@@ -30,11 +34,22 @@ @implementation WMFArticlePreviewTableViewCell
 
 #pragma mark - Setup
 
+- (void)blurAndShowLoadingIndicator {
+    self.blurView.hidden = NO;
+    [self.activityIndicator startAnimating];
+}
+
+- (void)unblurAndHideLoadingIndicator {
+    self.blurView.hidden = YES;
+    [self.activityIndicator stopAnimating];
```

Comment:
right, just got to that part :smile: you could probably omit the `start`/`stopAnimating` and let the activity indicator always animate when shown.